tools/SetupFlow/DevHome.SetupFlow/Controls/PackageDetailsTooltip.xaml.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation and Contributors
+// Licensed under the MIT license.
+
+using DevHome.SetupFlow.ViewModels;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+
+namespace DevHome.SetupFlow.Controls;
+
+public sealed partial class PackageDetailsTooltip : ToolTip
+{
+    public PackageViewModel Package
+    {
+        get => (PackageViewModel)GetValue(PackageProperty);
+        set => SetValue(PackageProperty, value);
+    }
+
+    public PackageDetailsTooltip()
+    {
+        this.InitializeComponent();
+    }
+
+    private static readonly DependencyProperty PackageProperty = DependencyProperty.Register(nameof(Package), typeof(PackageViewModel), typeof(PackageDetailsTooltip), new PropertyMetadata(null));

```

Comment:
For these I tend to prefer not to have each param on its own line, since if there are multiple properties, it's easier to read looking down the columns of parameters. Maybe compromise is putting a newline around the = so it's shorter, but still easy to read?

Reply:
@bbonaby I agree with your suggestion as a general rule. However, as @krschau, for `DependencyProperty` we may have multiple ones and they all have the same structure